components/BreadCrumb.js

```diff
@@ -0,0 +1,43 @@
+import Link from '@components/Link'
+import { FiChevronRight } from 'react-icons/fi'
+import {AiOutlineHome} from 'react-icons/ai'
+
+export default function BreadCrumb({pageData}) {
+
+  return (
+    <nav className="flex  p-3 " aria-label="Breadcrumb">
```

Comment:
I am not sure if we need `nav` here, or if we only use it for the main navigation 🤔 - I will ask in EddieHub Discord

Btw the spacing doesn't look quite right
```suggestion
    <nav className="flex p-3" aria-label="Breadcrumb">
```